alphastats/gui/utils/overview_helper.py

```diff
@@ -12,13 +13,18 @@ def get_sample_histogram_matrix(user_session_id):
     return st.session_state.dataset.plot_samplehistograms()
 
 
+@st.cache_data
+def get_intensity_distribution_unprocessed(user_session_id):
+    return st.session_state.dataset.plot_sampledistribution(use_raw=True)
+
+
 @st.cache_data
 def get_intensity_distribution_processed(user_session_id):
     return st.session_state.dataset.plot_sampledistribution()
 
 
 @st.cache_data
-def get_display_matrix(user_session_id):
+def get_display_matrix():
```

Comment:
we need at least one argument for every cached funciton or the output will never change.

Reply:
ah, thanks, got it